submissions/LGeorge007/friends-app/script.js

```diff
@@ -0,0 +1,162 @@
+const content = document.querySelector(".content");
+const navPanel = document.querySelector(".sidebar");
+const button = document.querySelector(".menu-button");
+const form = document.querySelector('.filters');
+const searchInput = document.getElementById('search-input');
+const resetButton = document.getElementById('reset-button');
+const url = 'https://randomuser.me/api/?results=120&inc=gender,name,location,email,phone,picture,dob&appid=YI09-4F4Y-E9IS-OTSH';
+const state = {
+    gotResult: [],
+    filterBySex: "",
+    filterByName: "",
+    sortByAge: "",
+    sortByName: "",
+    toDefault(arr) {
+        this.filterBySex = "all";
+        this.filterByName = "",
+        this.sortByAge = "ascending";
+        this.sortByName = "none";
+        this.gotResult = arr;
+    }
+};
+let result = [];
+
+const sortByAge = function(a, b) {
+    return a.dob.age == b.dob.age
+                ? 0
+                : a.dob.age <= b.dob.age
+                        ? -1
+                        : 1;
+}
+
+const sortByName = function(a, b) {
+    return a.name.first.toLowerCase() == b.name.first.toLowerCase()
+                ? 0
+                : a.name.first.toLowerCase() >= b.name.first.toLowerCase()
+                        ? 1
+                        : -1;
+}
```

Comment:
You can use .localeCompare(), the function returns exactly 0 1 -1. And you don't need .toLoweCase()
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare